handbook/engineering/code_reviews.md

```diff
@@ -42,31 +42,32 @@ It should be obvious to any other engineer on the team from the PR description a
 - "I don't have someone to review this code"
-    - Ask for help to identify someone else on the team with whom you can share your knowledge, context, and ownership.
+  - Ask for help to identify someone else on the team with whom you can share your knowledge, context, and ownership.
 
 If we see that there are too many [PRs being merged without approval](https://github.com/pulls?page=1&q=is%3Apr+org%3Asourcegraph+is%3Amerged+review%3Anone+-author%3Aapp%2Frenovate&utf8=%E2%9C%93) that lack an appropriate reason, then we will have to consider an automated solution to enforce code reviews.
 
 ## What makes an effective Pull Request (PR)?
 
-An effective PR minimizes the amount of effort that is required for the reviewer to understand your change so that they can provide high quality feedback in a timely manner. [Futher reading](https://www.atlassian.com/blog/git/written-unwritten-guide-pull-requests).
+An effective PR minimizes the amount of effort that is required for the reviewer to understand your change so that they can provide high quality feedback in a timely manner. [Further reading](https://www.atlassian.com/blog/git/written-unwritten-guide-pull-requests).
 
 Do:
 
 - Prefer small PRs. If a large PR is unavoidable, try to organize it into smaller commits (with [good commit messages](commit_messages.md)) that are reviewable independently (and indicate this to the reviewer in the description).
 - Create a draft PR first and review your own diff as if you were reviewing someone else's change. This helps you empathize with your reviewer and can help catch silly mistakes before your reviewer sees them (e.g. forgetting to `git add` a file, forgetting to remove debugging code, etc.).
-- Create meaningful PR title and description that communicates **what** the PR does and **why** (the **how** is the diff). 
-    - Include links to relevant issues (e.g. "closes #1234", "helps #5678").
-    - Include a screenshot, GIF, or video if you are making user facing changes.
-    - Discuss alternative implementations that you considered but didn't pursue (and why).
-    - Describe any remaining open questions that you are seeking feedback on.
+- Create meaningful PR title and description that communicates **what** the PR does and **why** (the **how** is the diff).
+  - Include links to relevant issues (e.g. "closes #1234", "helps #5678").
+  - Include a screenshot, GIF, or video if you are making user facing changes.
+  - Discuss alternative implementations that you considered but didn't pursue (and why).
+  - Describe any remaining open questions that you are seeking feedback on.
 - Make it clear who's review(s) you are waiting for.
 - Politely remind your reviewer that you are waiting for a review if they haven't responded after one business day.
+- Don't force push commits that erase the latest reviewed commit or any commit prior to that. This makes it easy for reviewers to pick up where they left off in the previous round.
```

Comment:
```suggestion
- If you need to work on your PR after a first round of reviews, try not to force push commits that erase the latest reviewed commit or any commit prior to that. This makes it easy for reviewers to pick up where they left off in the previous round.
```

Reply:
I think @mrnugget wording is clearer.

Reply:
I like this better because it also includes `try not to` which implies there are some exceptional cases one could run into in which it is OK to do so.